src/librustc/mir/visit.rs

```diff
@@ -222,7 +222,7 @@ macro_rules! make_mir_visitor {
             }
 
             fn visit_closure_substs(&mut self,
```

Comment:
Seems like we can remove this visit method altogether, I think? Based on my quick ripgrep of the uses, I think they could be removed (and calls redirected to `visit_substs`